### DISCUSSION TITLE
How to concatenate strings?

### DISCUSSION THREAD

**Discussion by linux-time** [vectordotdev/vector#21538] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/de071c58-0754-4bbb-be60-c10e4290336d)
What is the correct way to concatenate .message and .foo in the image and assign them to .test? Thanks!

**Comment by jszwedko** [comment]
`.test = .message + .foo`

**Reply by linux-time** [reply, marked as answer]
I tried .test = .message + .foo will report an error. You need to write it like this: .test = string!(.message) + string!(.foo)
